pod/main/utils.py

```diff
@@ -0,0 +1,2 @@
+def is_ajax(request) -> bool:
```

Comment:
Merci d'ajouter un entete pydoc sur chaque nouvelle fonction ("""Check that the request is made by javascript""" par exemple)